### DISCUSSION TITLE
NotebookApp 404 GET /json errors

### DISCUSSION THREAD

**Discussion by bugsbunny88** [microsoft/vscode-jupyter#11114] [Questions and Answers]
Hey guys,
as of 1-2 weeks ago this started happening, and I'm starting to lose my mind trying to figure out how to resolve this. If someone could have a look at let me know how to even start figuring this out....
![image](https://user-images.githubusercontent.com/24847798/184519145-cea90b55-e4d6-4000-878e-7a25a7f2cbc2.png)
![image](https://user-images.githubusercontent.com/24847798/184519156-2dc9e574-3311-411f-be41-fee3a0d52eff.png)

**Comment by DonJayamanne** [comment]
Not really sure what this is, I've never seen this before. Fortunately its a warning that can be ignored.
Though i'm not sure what it is.
Do you know when you get this warning message, or what you are doing when you get this.
Is it possible you are trying to access the Jupyter server manually outside vscode.
Also, could you provide all of the output from the `Jupyter` output panel

**Comment by bugsbunny88** [comment]
Well essentially jupyter just hangs , for minutes at a time with continuous errors like shown above is why I thought this was the core issue, its the hanging part thats the issue I want to resolve. Whether in the background its littered with errors I'm fine with .

**Comment by DonJayamanne** [comment]
>  the hanging part thats the issue I want to resolve

thanks, that information is useful.
please could you file an issue with the details requested in the issue template and I'll help you get unblocked.
